### DISCUSSION TITLE
Cannot import disk numbers greater than 127

### DISCUSSION THREAD

**Discussion by strange-antiquark** [enzo1982/freac#357] [Help & Support] [answered]
I just grabbed the 2016 200-CD box set of the complete works of Mozart off Usenet. I have determined (by a small random sample) that every file in the set (there are over 3300 of them) has the disc number and disc total in the header information.

Problem: freac will not import disc numbers larger than 127. Is this being treated as a signed byte? If so, should it not be unsigned, as it is hardly possible to have a negative number of discs?

**Comment by strange-antiquark** [comment, marked as answer]
At least as of the continuous build of 08/18, disk numbers > 127 are now being imported, so I am marking this one resolved.
Many many many many many thanks :D

**Reply by enzo1982** [reply]
Yes, I changed the data field to unsigned recently. Thank you for bringing this up!

I'll also allow three digits in the tag editor fields. Coming soon...
